gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextSection.kt

```diff
@@ -0,0 +1,238 @@
+                            scope.launch {
+                                snackState.showQESnackbar(
+                                    message = context.getString(action.type.errorStringRes),
+                                    snackbarType = SnackbarType.Error,
+                                )
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    GravatarTheme {
+        Box(modifier = modifier.wrapContentSize()) {
+            state.altTextSectionUiState?.let { altTextState ->
+                AltTextSection(
+                    altTextState = altTextState,
+                    onEvent = viewModel::onEvent,
+                )
+                QESnackbarHost(
+                    modifier = Modifier
+                        .align(Alignment.BottomStart),
+                    hostState = snackState,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+internal fun AltTextSection(
+    altTextState: AltTextSectionUiState,
+    onEvent: (AvatarPickerEvent) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Surface(modifier = modifier.fillMaxWidth()) {
+        Box(
+            modifier = Modifier
+                .border(
+                    width = 1.dp,
+                    color = MaterialTheme.colorScheme.surfaceContainerHighest,
+                    shape = RoundedCornerShape(8.dp),
+                )
+                .padding(16.dp),
+        ) {
+            Column {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier.fillMaxWidth(),
+                ) {
+                    QESectionTitle(
+                        title = stringResource(R.string.gravatar_qe_avatar_alt_text_section_title),
+                        modifier = Modifier,
+                    )
+                    Text(
+                        text = stringResource(id = R.string.gravatar_qe_avatar_alt_text_section_what_is),
+                        color = MaterialTheme.colorScheme.primary,
+                    )
+                }
+
+                Row(
+                    verticalAlignment = Alignment.Top,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 16.dp),
+                ) {
+                    val cornerRadius = 8.dp
+                    val avatarSize = 96.dp
+                    val sizePx = with(LocalDensity.current) { avatarSize.roundToPx() }
+                    AsyncImage(
+                        model = altTextState.avatar.imageUrlWithSize(sizePx),
+                        contentDescription = stringResource(
+                            id = R.string.gravatar_qe_selectable_avatar_content_description,
+                        ),
+                        modifier = Modifier
+                            .size(96.dp)
```

Comment:
```suggestion
                            .size(avatarSize)
```

Shouldn't it be that?